### ISSUE TITLE
Add github action for testing cmake installation

### ISSUE THREAD

**Issue by ncesario-lunarg** [KhronosGroup/glslang#3462]
Add a test that exercises cmake's `find_package` for all installed targets. These currently include:
- [ ] glslang::OSDependent
- [ ] glslang::glslang
- [ ] glslang::MachineIndependent
- [ ] glslang::GenericCodeGen
- [ ] glslang::glslang-default-resource-limits
- [ ] glslang::glslang-standalone
- [ ] glslang::spirv-remap
- [ ] glslang::SPVRemapper
- [ ] glslang::SPIRV

**Comment by juan-lunarg** [comment]
For reference I have added testing for find_package in the Vulkan-Loader. It's run via `ctest`:
https://github.com/KhronosGroup/Vulkan-Loader/pull/1423

This makes it trivial to test.

**Comment by juan-lunarg** [comment]
` glslang::OSDependent` I'd argue should NOT be installed. It's useless to end users. Making it an object library will prevent it from getting installed on users machines. While still keeping the abstraction.

**Comment by arcady-lunarg** [comment]
Yes, OSDependent, MachineIndependent, and GenericCodeGen are not for end user use and should be object libraries.

**Comment by jpr42** [comment]
After looking into this further it seems  glslang::SPIRV is also another stub library.
